tests/test_converter.py

```diff
@@ -551,36 +545,60 @@ def test_operation_transformed_into_qubit_unitary(self, recorder):
         assert np.array_equal(recorder.queue[0].parameters[0], ex.CHGate().to_matrix())
         assert recorder.queue[0].wires == Wires([0, 1])
 
-    def test_quantum_circuit_error_by_passing_wrong_parameters(self, recorder):
-        """Tests the load method for a QuantumCircuit raises a QiskitError,
-        if the wrong type of arguments were passed."""
+    def test_qiskit_gates_to_be_deprecated(self, recorder):
```

Comment:
This is a good idea. This way, we will know when the gates have been removed due to this test failing :)